topics/mobile/index.md

```diff
@@ -0,0 +1,8 @@
+---
+aliases: mobile-apps
+display_name: Mobile
+short_description: Tools to improve your workflow for the small screen.
```

Comment:
I think this short description is still too focused on Marketplace / one particular use case. If we search for repos tagged [`#mobile`](https://github.com/topics/mobile?o=desc&s=updated) we'll see full blown apps, games, operating systems, databases, etc. They would quite fit this description...

![screen_shot_2018-06-23_at_11_01_52_am](https://user-images.githubusercontent.com/121322/41869895-54441468-786f-11e8-99b1-6a2f5a45a215.png)

Maybe make the description overly generic, remove it, or make a few more granular options e.g. `#mobile-app-development.md`, `mobile-framework.md`?